src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import { FC, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+
+interface Props {
+  todo: Todo;
+  isLoading: boolean;
+  onTodoDelete?: (todoId: number) => void;
+}
+
+export const TodoItem: FC<Props> = ({ todo, isLoading, onTodoDelete }) => {
+  const [isEdited /*setIsEdited*/] = useState(false);
```

Comment:
Why did you comment your setter?

Reply:
I am currently using isEdited for conditional rendering, and i expect to use setter in future for updating